public_html/admin-v2/races/js/results.js

```diff
@@ -29,31 +27,48 @@ const results = new Vue({
     availableWinHorses: function () {
       const placeId = this.enteredResults.place;
       const showId = this.enteredResults.show;
-      return this.horses.filter(horse => horse.id != placeId && horse.id != showId);
+      return this.horses.filter(
+        (horse) => horse.id != placeId && horse.id != showId
+      );
     },
     availablePlaceHorses: function () {
       const winId = this.enteredResults.win;
       const showId = this.enteredResults.show;
-      return this.horses.filter(horse => horse.id != winId && horse.id != showId);
+      return this.horses.filter(
```

Comment:
Can you create util.js file and encapsulate this?? So we can reuse it in multiple files.

Reply:
The filter statement is slightly different in all 3 computed properties. 

Also the logic needs to be inside the computed property or their may be some errors
https://vuejs.org/v2/guide/computed.html